app/services/api/v3/participant_declarations_query.rb

```diff
@@ -52,7 +52,7 @@ def participant_declarations_from(paginated_join)
         ActiveRecord::Associations::Preloader.new(
```

Comment:
Is it worth testing theses more specific eager loads? we don't for others, but we could with something like:

```
expect(npq_declarations.map { |d| d.association(:outcomes) }).to all(be_loaded)
```

Reply:
added, TIL you could test that